src/Makefile.m32

```diff
@@ -238,13 +238,20 @@ endif
 ifeq ($(findstring -ngtcp2,$(CFG)),-ngtcp2)
 NGTCP2 = 1
 endif
+ifeq ($(findstring -unicode,$(CFG)),-unicode)
+UNICODE = 1
+endif
 
 INCLUDES = -I. -I../include -I../lib
 ifdef SSL
   ifdef WINSSL
     CFLAGS += -DCURL_WITH_MULTI_SSL
   endif
 endif
+ifdef UNICODE
+  CFLAGS += -DUNICODE -D_UNICODE
+  LDFLAGS += -municode
```

Comment:
Actually, `-municode` is a linker *and* a compiler option. It currently only makes sure that `UNICODE` is defined, but I added it to CFLAGS too in the CMake version of this for consistency.

Reply:
Oh, right, I did not know this. Yet, I'd prefer to go with "classic" compiler flags for portability and transparency.

Reply:
Yes, in the other build systems we have both the preprocessor defines and the compiler and linker option (the compiler flag doesn't define `_UNICODE`, so that's required anyway). I don't see a portability reason not to use the compiler option if the linker option is used as either both or none are supported.